examples/async-universal/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": ["es2015", "react"],
+  "plugins": ["transform-class-properties"], // needed to get local context.router
```

Comment:
We don’t want to have anything later than ES2015 in the examples. Just change the code to assign static properties instead:

```js
class MyComponent extends Component {}
MyComponent.contextTypes = ...
```